app/views/teams/move_to_business_group.html.slim

```diff
@@ -0,0 +1,13 @@
+- content_for :page_title do
+  = t('page_title.move_directorate_page', team_name: @team.name)
```

Comment:
Can we please be specific with the page titles as it's one of the things we have tickets for in the Offender SAR journey for Accessibility. Let's maybe specify 'Move..' and 'Confirm Move...' or something.

Reply:
The current page title is like e.g. the name of directorate  is ABC
`Moving a directorate - ABC - Track-a-query`
so you suggest to change it to 
`Move ABC - Track-a-query` ?

Reply:
Sorry I mean Move a directorate - ABC - Track-a-query 
and then Confirm Move directorate - ABC - Track-a-query

Reply:
I see,  thank you!, done